src/odemis/gui/cont/tabs.py

```diff
@@ -1796,6 +1796,15 @@ def __init__(self, name, button, panel, main_frame, main_data):
             calibrations
         )
 
+        # Controller for acquistion settings panel
+        self._acq_settings_controller = settings.SettingsController(
+            panel.pnl_acq_settings,
+            "dwell time"
```

Comment:
That's the "default message", which is shown if no setting is available. `""` would be better here.